internal/datacoord/segment_manager.go

```diff
@@ -53,25 +53,25 @@ type Manager interface {
 	GetFlushableSegments(ctx context.Context, channel string, ts Timestamp) ([]UniqueID, error)
 	// UpdateSegmentStats update segment status
 	UpdateSegmentStats(stat *internalpb.SegmentStatisticsUpdates)
+	// ExpireAllocations notify segment status to expire old allocations
+	ExpireAllocations(channel string, ts Timestamp) error
 }
 
+// segmentStatus stores allocation entries and temporary row count
 type segmentStatus struct {
 	id             UniqueID
-	collectionID   UniqueID
-	partitionID    UniqueID
-	sealed         bool
-	total          int64
-	insertChannel  string
 	allocations    []*allocation
 	lastExpireTime Timestamp
```

Comment:
SegmentInfo has lastExpireTime field, so we dont have to keep lastExpireTime in segmentStatus

Reply:
`lastExpireTime` field is used in channelSealPolicy